CTP/doc/sql/sql_guide.md

```diff
@@ -0,0 +1,450 @@
+# 1 Test Objective
+SQL is an ANSI (American National Standards Institute) standard computer language, but there are still many different versions of SQL.However, in order to be compatible with ANSI standards, they must all support some major commands (such as SELECT, UPDATE, DELETE, INSERT, WHERE, and so on) in a similar way.  
+CUBRID SQL detection, SQL detection for each build to detect the basic functions of the database and ensure the correct output of commands.
+
+# 2 Test Deployments and Introduction
+To facilitate the execution of various test requirements, CUBRID QA has developed a range of automated tools. This chapter describes the environment construction.
```

Comment:
Please remove "a range of automated tools". It's useless to reader. 
Only focus on CTP guide.